lib/vertica/column.rb

```diff
@@ -34,10 +34,10 @@ class Column
       7   => [:float,        FLOAT_CONVERTER],
       8   => [:char,         STRING_CONVERTER],
       9   => [:varchar,      STRING_CONVERTER],
-      10  => [:date,         lambda { |s| Date.new(*s.split("-").map{|x| x.to_i}) }],
+      10  => [:date,         lambda { |s| Date.new(*s.split("-").map(&:to_i)) }],
       11  => [:time,         nil],
-      12  => [:timestamp,    lambda { |s| DateTime.parse(s, true) }],
-      13  => [:timestamp_tz, lambda { |s| DateTime.parse(s, true) }],
+      12  => [:timestamp,    lambda { |s| Time.parse(s) }],
+      13  => [:timestamp_tz, lambda { |s| Time.parse(s) }],
```

Comment:
Yup. This will be a major breaking release due to other API changes anyway (see #47). Note that handling timestamps differently than before is also something that will potentially break existing applications as well